setup.py

```diff
@@ -49,7 +48,9 @@ def read(fname):
 
 extras_require = {
     'docs': ['sphinx'],
-    'tests': test_requires
+    'tests': test_requires,
+    ': implementation_name != "pypy"': ['psycopg2'],
```

Comment:
This requires setuptools at least at v 21 and pip 9 to be able to install it with pip (older pip versions just ignore that)